Makefile

```diff
@@ -114,35 +108,36 @@ pytest: compose-build
 	@docker-compose run $(IMAGE_NAME) \
 		pytest $(pytestpath) -m "not integration and not integration_erasure and not integration_external"
 
-# Run the pytest integration tests.
-pytest-integration-access: compose-build
-	@echo "Building additional Docker images for integration tests..."
-	@docker-compose -f docker-compose.yml -f docker-compose.integration-test.yml build
-	@echo "Bringing up the integration environment..."
-	@docker-compose -f docker-compose.yml -f docker-compose.integration-test.yml up -d
-	@echo "Waiting 20s for integration containers to be ready..."
-	@sleep 20
-	@echo "Running additional setup for mssql integration tests"
-	@docker exec fidesops python tests/integration_tests/mssql_setup.py
-	@echo "Running pytest integration tests..."
-	@docker-compose -f docker-compose.yml -f docker-compose.integration-test.yml \
-		run $(IMAGE_NAME) \
-		pytest $(pytestpath) -m integration
-	@docker-compose -f docker-compose.yml -f docker-compose.integration-test.yml down --remove-orphans
+configure-infrastructure: compose-build
```

Comment:
can we add this to docs? In what scenario would we use this command?

Reply:
This is doing the same thing as `make integration-shell` so I'll remove it

Reply:
I see in `make integration shell` we're additionally using `open_shell=true`, so as long as we'll never need to setup the venv without the shell, as we do here

Reply:
We aren't currently doing that, if we do need to set it up without the shell we can always run the command then exit the shell